km52/Larionova_Yuliia/laboratory1.sql

```diff
@@ -1,2 +1,135 @@
 -- LABORATORY WORK 1
 -- BY Larionova_Yuliia
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+вставляти дані до таблиць
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+create user larionova identified by pass
```

Comment:
use capital letters

Reply:
It's not necessary  to use capital letters in SQL.